app/main.py

```diff
@@ -1,25 +1,90 @@
 from abc import ABC
+from typing import Union, Type
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(
+            self,
+            owner: Union["SlideLimitationValidator", "Visitor"],
+            name: str
+    ) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(
+            self,
+            instance: Union["SlideLimitationValidator", "Visitor"],
+            owner: Union["SlideLimitationValidator", "Visitor"]
+    ) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(
+            self,
+            instance: Union["SlideLimitationValidator", "Visitor"],
+            value: int
+    ) -> None:
+        if isinstance(value, int):
+            if self.min_amount <= value <= self.max_amount:
+                setattr(instance, self.protected_name, value)
+                return
+            raise ValueError
```

Comment:
```suggestion
       raise ValueError
```
Wrong tabulation. Also add description for exception